### PR TITLE
Limit TaskList polling, fixup build display

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
@spadgett review

**Comment by liggitt** [comment]
@csrwng can you take a look at this? TaskList was polling continuously... I don't think we actually need to poll

**Comment by liggitt** [comment]
Should help with #1921

File: assets/app/scripts/services/tasks.js [hunk 1]
```diff
@@ -30,49 +29,45 @@ angular.module('openshiftConsole')
...
       }
-      else if (task.status === "completed" && !task.hasErrors) {
-        // Clear out successfully completed tasks after a timeout
-        if ((Date.now() - task.completedTime) > TASK_TIMEOUT) {
-          task.toDelete = true;
+
+      // Otherwise, queue the message to be deleted
+      // $timeout handles calling us in an apply loop
+      $timeout(function() {
+        task.toDelete = true;
```
Comment (csrwng) [inline assets/app/scripts/services/tasks.js]: it may be better to do something like:
```
i = tasks.indexOf(task)
if (i != -1) { tasks.splice(i,1); }
```

**Comment by csrwng** [comment]
other than one comment, LGTM

File: assets/app/scripts/app.js [hunk 2]
```diff
@@ -189,12 +189,13 @@ angular
     }
   }])
   .run(function($interval, dateRelativeFilter, durationFilter) {
-    $interval(function() {
+    // Use setInterval instead of $interval because we're directly manipulating the DOM and don't want scope.$apply overhead
+    setInterval(function() {
```
Comment (spadgett) [inline assets/app/scripts/app.js]: Should we still use `$interval` and pass `false` for the `invokeApply` param?

https://docs.angularjs.org/api/ng/service/$interval

File: assets/app/scripts/app.js [hunk 3]
```diff
@@ -189,12 +189,13 @@ angular
     }
   }])
   .run(function($interval, dateRelativeFilter, durationFilter) {
```
Comment (spadgett) [inline assets/app/scripts/app.js]: Should be able to remove the `$interval` param.

**Comment by spadgett** [comment]
LGTM, much better.

**Comment by liggitt** [comment]
[merge]

**Pull request merged**